### PR TITLE
[Form] Improve Persian (Farsi) Translation For Forms

### PR DESCRIPTION
**Pull request by mostafazs**
| Q             | A
| ------------- | ---
| Branch?       | 6.1 
| Bug fix?      | no
| New feature?  | no 
| Deprecations? | no 
| License       | MIT

Improve Persian (Farsi) Translation For Form Component Translation File.

### CODE REVIEW

File: src/Symfony/Component/Form/Resources/translations/validators.fa.xlf [hunk 1]
```diff
@@ -24,7 +24,7 @@
             </trans-unit>
             <trans-unit id="101">
                 <source>The selected choice is invalid.</source>
-                <target>گزینه‌ی انتخاب‌شده نامعتبر است.</target>
+                <target>گزینه‌ انتخاب‌شده نامعتبر است.</target>
```
Comment (BahmanMehrdad) [inline src/Symfony/Component/Form/Resources/translations/validators.fa.xlf]: This text should be replaced with this sentence (There must be a space between the two words `انتخاب‌` and `شده`). The rest of the edited sentences are correct:
`<target>گزینه‌ انتخاب‌ شده نامعتبر است.</target>`

**Review by BahmanMehrdad — CHANGES_REQUESTED** [review]

File: src/Symfony/Component/Form/Resources/translations/validators.fa.xlf (hunk 1 above)
Comment (mostafazs) [reply src/Symfony/Component/Form/Resources/translations/validators.fa.xlf]: Yes this is also true

**Review by mostafazs — COMMENTED** [review]
Correct

**Review by azjezz — APPROVED** [review]

**Review by BahmanMehrdad — APPROVED** [review]

**Comment by azjezz** [comment]
This should be considered a bug fix, and merged in 5.4 i suppose.

**Comment by mostafazs** [comment]
> This should be considered a bug fix, and merged in 5.4 I suppose.

@azjezz so I must remove this PR and create a new one on the 5.4 repo? I don't know that 6.1 is just for new features.

**Comment by fancyweb** [comment]
All translations are actually backported to the lowest maintained branch (4.4 atm) so you can just change the target branch to 4.4 and there shouldn't be any conflict.

**Review by wouterj — APPROVED** [review]

**Comment by wouterj** [comment]
Thank you @mostafazs!

As you created the PR on the 6.1 branch of your fork, I took your commit and created a new PR to 4.4: #44591 We'll merge it to 6.1 from there.

For your next contribution, it is recommended that you create a new branch for each PR (see [the contributing docs](https://symfony.com/doc/current/contributing/code/pull_requests.html#create-a-topic-branch)). A PR is always between a branch in your fork and the main repository. Having a separate branch for each PR allows to work on more than 1 PR at the same time. And it makes sure you can freely change the base version without changing the 6.1 branch in your fork.

However, as I only took your commit, you're still the full author of this change. All credits (and badges) to you and hope to see you back another time!

**Pull request closed without merging**